### PR TITLE
modified profile screen to fit personal account

### PR DESCRIPTION
**Pull request by Vivian-Xiao1025**
Description: modified profile screen to fit personal account

Tests Performed: n/a

Trello Card Link: https://trello.com/c/JgjrEmPi

Screen Shot 2023-01-21 at 3 03 05 PM

### CODE REVIEW

**Comment by shreyasminocha** [comment]
Can we adjust this to reduce duplication between the profile and account pages? I suggest moving shared components into separate files, and then importing and using them on both pages.

File: lib/screen/personal_account_screen.dart [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  @override
+  void dispose() {
+    _tabController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // bottomNavigationBar: const MyNavigationBar(),
```
Comment (baylee-rl) [inline lib/screen/personal_account_screen.dart]: Remove commented out lines

File: lib/screen/profile_screen.dart [hunk 2]
```diff
@@ -33,7 +37,7 @@ class _ProfileScreenState extends State<ProfileScreen>
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-      bottomNavigationBar: const MyNavigationBar(),
+      // bottomNavigationBar: const MyNavigationBar(),
```
Comment (baylee-rl) [inline lib/screen/profile_screen.dart]: More commented out lines

File: lib/screen/personal_account_screen.dart [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+                text: "Edit Profile",
+                onPressed: () => {},
+                toggle: false,
+              ),
+              const SizedBox(height: 10),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceAround,
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: [
+                  Column(
```
Comment (baylee-rl) [inline lib/screen/personal_account_screen.dart]: [Create followers/following widget.](https://github.com/rice-apps/tunedin/pull/23/files/d4620f4d824072c82404467aefab55cb3be33769#r1083332482)

**Review by baylee-rl — CHANGES_REQUESTED** [review]
looks great! just a few small comments - i'll go ahead and make them since they overlap w/ #23 and then get both PRs merged

**Pull request merged**